src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -869,6 +869,7 @@ private CalculableSentenceEnvelope determineCalculableSentenceEnvelope(OffenderB
         final var person = new uk.gov.justice.hmpps.prison.api.model.calculation.Person(
             offenderBooking.getOffender().getNomsId(),
             offenderBooking.getOffender().getBirthDate(),
+            offenderBooking.getOffender().getLastName(),
```

Comment:
note that in Prison API we would normally try and capitalise as well e.g. https://github.com/ministryofjustice/prison-api/blob/e7942ead7a4316e7d150d644241540d011ba1f42/src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java#L179

Reply:
@petergphillips, ok I've updated